main.py

```diff
@@ -39,65 +39,64 @@ def _run(self):
         for submission in self.reddit.subreddit(self.config.SUBREDDIT).stream.submissions():
             if self.database.known_submission(submission) is True:
                 continue
+            else:
+                self.database.save_submission(submission)
```

Comment:
Not sure we actually want this here specifically.  I noticed we had a few edge cases where we were not saving the submission so I moved this up to here but now I realize that if a service fails we will have saved that submission and skip it the next time.

I'll have to look into how PRAW works further.  Is it even possible it gives us the same submission more than once?

Reply:
By default it will get the last 100 submissions and stream new posts

Reply:
Looking at it now it might the edge case might have been this:

https://github.com/martijnboers/BlotterTrax/pull/13/files#diff-5bc02cefb3ea9e27f1a6776eabd1935dR70

I prefer not saving here and explicitly calling the save method following all possible paths because of the reason you mention

Reply:
Yep.  I have since merged that into this branch so the above code snippet is outdated/unused.